geth/api/backend_txqueue_test.go

```diff
@@ -53,22 +51,25 @@ func (s *BackendTestSuite) TestSendContractTx() {
 
 			// the first call will fail (we are not logged in, but trying to complete tx)
 			log.Info("trying to complete with no user logged in")
-			txHash, err = s.backend.CompleteTransaction(event["id"].(string), TestConfig.Account1.Password)
+			txHash, err = s.backend.CompleteTransaction(
+				common.QueuedTxID(event["id"].(string)), TestConfig.Account1.Password)
```

Comment:
Overall I agree, it's easier to read and edit. However, in my opinion, it's fine if the last argument is a message like in this example:
```go
t.EqualError(err, txError.Error(), 
  fmt.Sprintf("some long message with formating: %v", tx))
```

After coding in Python for a few years, I might be biased though :P

Reply:
I wouldn't say so. When you introduce a new nested indenting level without a closing structure on the initial indenting level, you can't quickly locate the end of the function call, your eyes are looking below and below searching for the closing parenthesis. And in case of having each argument on a separate line you can quickly figure out where your function call ends. You also don't guess if it's an indenting error or by design.

By the way, Go doesn't enforce line width and keeping long lines is sometimes more readable than splitting them.

Reply:
I think it's difficult to say for sure that it's the way everyone's eyes work. As I said, this type of code in some other languages and environments, like Python, is something normal. 

Consistency and project standards are more important and if we decided to put each param in a line line in such a case, then we should enforce it everywhere and I am willing to do so.

After we make the tests passing in CI, we can curate a list of linters and add `make lint` to the CI flow. It should handle such stylistic issues.

Reply:
It's not the rule yet but I'd like to get it a try and cancel it if most people say they don't like it.